io/src/test/java/org/commonjava/maven/ext/io/rest/BlacklistTranslatorTest.java

```diff
@@ -52,8 +54,11 @@ public void before()
     {
         LoggerFactory.getLogger( BlacklistTranslatorTest.class ).info( "Executing test " + testName.getMethodName() );
 
+        Map<String, String> headers = new LinkedHashMap<>( 1 );
+        headers.put( "Foo", "bar" );
```

Comment:
I am trying to add a fake header to test that it's passed through. It can be seen in the logs (of the base class), but I don't think there's any way to get at the headers and assert something here.

Reply:
`HttpHeaderHeaderTest` hacks around it. Also `RESTParametersTest` creates a dummy `AbstractHandler` which could also be used (as the headers can be extracted from `HttpServletRequest`)

Reply:
OK, I created a new test based on `RESTParametersTest`.